test.sh

```diff
@@ -40,11 +35,13 @@ touch data/fn.db data/fn.mq
 # start fn
 CONTAINER_ID=$($fn start -d)
 
+TEST_API_URL="localhost:8080"
```

Comment:
that actually would not work with remote docker API.

Reply:
I've updated this behaviour to honor FN_API_URL again and added some code to generate variances of it to check URL handling, thanks